Packs/Core/Playbooks/playbook-Ransomware_Response.yml

```diff
@@ -0,0 +1,1399 @@
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 1140,
+          "y": 3540
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "138":
+    id: "138"
+    taskid: 5b75d413-1efc-4a67-8d38-b54073d7d67e
+    type: playbook
+    task:
+      id: 5b75d413-1efc-4a67-8d38-b54073d7d67e
+      version: -1
+      name: Ransomware Advanced Analysis
+      description: |-
+        This playbook is responsible for detecting the ransomware type and searching for available decryptors.
+
+        The playbook uses the ID-Ransomware service, which allows detecting the ransomware using multiple methods.
+      playbookName: Ransomware Advanced Analysis
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "63"
+    separatecontext: false
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 720,
+          "y": 4150
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "121_131_yes": 0.71,
+      "121_63_#default#": 0.19,
+      "125_126_#default#": 0.51,
+      "126_124_yes": 0.53,
+      "134_130_False Positive": 0.8,
+      "134_3_True Positive": 0.68,
+      "5_103_yes": 0.54,
+      "5_136_yes": 0.78,
+      "5_14_#default#": 0.54
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3785,
+        "width": 1780,
+        "x": -260,
+        "y": 2350
+      }
+    }
+  }
+inputs:
+- key: earlyRemediation
+  value:
+    simple: "True"
+  required: false
+  description: Whether to execute the early remediation phase.
+  playbookInputQuery:
+- key: AutoContainment
+  value:
+    simple: "True"
+  required: false
+  description: Whether to execute the containment actions automatically.
+  playbookInputQuery:
+- key: AutoEradication
+  value:
+    simple: "True"
+  required: false
+  description: Whether to execute the eradication actions automatically.
+  playbookInputQuery:
+- key: isolateRemoteAttacker
+  value:
+    simple: "True"
+  required: false
+  description: Whether to isolate the remote endpoint if the attack has been triggered
+    remotely.
+  playbookInputQuery:
+- key: isolateSimilarEndpoints
+  value:
+    simple: "True"
+  required: false
+  description: Whether to isolate the endpoints identified with similar IoCs to the
+    ransomware alert.
+  playbookInputQuery:
+- key: RunAdvancedAnalysis
+  value:
+    simple: "True"
+  required: false
+  description: Whether to execute the Ransomware Advanced Analysis playbook.
```

Comment:
lets add in the description that the Advanced Analysis playbook contains manual actions